src/mesh/boundary_nonuniform.hxx.in.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+from jinja2 import Environment
+
+env=Environment(trim_blocks=True);
+
+
+orders=range(2,5)
+boundaries=["Dirichlet","Neumann","Free"]
+
+header="""
+#include <utility>
+
+#include "boundary_op.hxx"
+
+// Define structs used
+struct Indices{
+  int x,y,z;
+};
```

Comment:
I used to pass it around, but then you complained. So that is why it is here.
I think having them in a struct is clarer then having just several integers.